src/Listeners/ReduceAttributeOptionValueListener.php

```diff
@@ -78,8 +78,8 @@ public function handle(EventInterface $event)
                 // iterate over the categories
                 foreach ($arts as $attributeOptionValues) {
                     foreach ($attributeOptionValues as $attributeOptionValue) {
-                        // load the attribute code
-                        $attributeCode = $attributeOptionValue[ColumnKeys::ATTRIBUTE_CODE];
+                        // Generate unique key for artefact to avoid duplicates
+                        $attributeCode = md5(\json_encode($attributeOptionValue));
```

Comment:
What does this fix?

Reply:
With the attribute check, only the first attribute value is imported.
However, the uniqueness (to avoid duplicate entries) is the complete line that is written into the artefact.